src/ui/controls/message-box/index.tsx

```diff
@@ -20,7 +21,7 @@ export default function MessageBox(props: MessageBoxProps) {
                     <Button class="message-box__button message-box__button-ok" onclick={props.onOK}>
                         OK
                     </Button>
-                    <Button class="message-box__button message-box__button-cancel" onclick={props.onCancel}>
+                    <Button class="message-box__button message-box__button-cancel" onclick={props.onCancel} style={props.hideCancel ? 'display: none' : ''}>
```

Comment:
This is JSX, we can simply not render this button. We don't need a hacky `display: none` CSS hack.